x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/trainedmodel/InferenceHelpers.java

```diff
@@ -72,4 +71,19 @@ public static String classificationLabel(double inferenceValue, @Nullable List<S
         }
         return classificationLabels.get(label);
     }
+
+    public static Double toDouble(Object value) {
```

Comment:
I know you already added unit test in TreeTests but it would be good to have at least one test (with a few calls) for this very method.